Packs/AWS-GuardDuty/Integrations/AWSGuardDuty/AWSGuardDuty.py

```diff
@@ -546,6 +546,9 @@ def main():
     timeout = params.get('timeout') or 1
     retries = params.get('retries') or 5
 
+    if demisto.args().get('region') is not None:
+        aws_default_region = demisto.args().get('region')
```

Comment:
It is better to leave the meaning of the variable name as it is.

Therefore, in lines 556-557 add like this:
```
region = demisto.args().get('region')
client = aws_client.aws_session(service=SERVICE, region=region)
```